packages/atomic/cypress/e2e/generated-answer.cypress.ts

```diff
@@ -174,6 +174,70 @@ describe('Generated Answer Test Suites', () => {
       });
     });
 
+    describe('when collapsible prop is provided', () => {
+      describe('answer height is more than 250px', () => {
+        const streamId = crypto.randomUUID();
+
+        const testTextDelta = generateLongTextAnswer();
+        const testMessagePayload = {
+          payloadType: 'genqa.messageType',
+          payload: JSON.stringify({
+            textDelta: testTextDelta,
+          }),
+          finishReason: 'COMPLETED',
+        };
+
+        beforeEach(() => {
+          mockStreamResponse(streamId, testMessagePayload);
+          setupGeneratedAnswerWithoutFirstIntercept(streamId, {
+            collapsible: true,
+          });
+        });
+
+        GeneratedAnswerAssertions.assertShowButton(true);
+        GeneratedAnswerAssertions.assertAnswerCollapsed(true);
+        GeneratedAnswerAssertions.assertShowMoreLabel(true);
+        GeneratedAnswerAssertions.assertFeedbackButtonsVisibility(false);
+        GeneratedAnswerAssertions.assertCopyButtonVisibility(false);
```

Comment:
I'm not sure we should hide the feeback and copy buttons...

What if somebody has found the answer they were looking for (or figured that the answer was not relevant to their question) in the first few sentences? Do we still want to force them to expand the answer to provide us with their feedback?

For both the feeback and copy buttons, suppose that the user has expanded (and read) the answer and then decides to collapse it again to have more vertical real estate to look at the result page. I don't think it makes sense to hide the buttons in this context either.

Reply:
This was UX requirements :/ maybe @adroletCoveo has more insights on this

Reply:
@VykaNguyen, as Audrey is not available for now, what do you think about this use case ? :)

Reply:
update: For now we will try it like this, and regarding the client feedbacks we could change this!

Reply:
> What if somebody has found the answer they were looking for (or figured that the answer was not relevant to their question) in the first few sentences? Do we still want to force them to expand the answer to provide us with their feedback?

Yes absolutely, perhaps the first few lines are not relevant but the rest is VERY relevant. Also how valuable is the feedback of someone who didn't even bother to expand the answer to read more than the first lines?

> For both the feeback and copy buttons, suppose that the user has expanded (and read) the answer and then decides to collapse it again to have more vertical real estate to look at the result page. I don't think it makes sense to hide the buttons in this context either.

It would make it much more complex if we had to track if the user has ever expanded the answer, but then decided to collapse it, to still allow them to give us feedback. To gain probably little to no value for the few who would expand > collapse > want to give feedback after this...